src/codegen/compile_enhanced.ml

```diff
@@ -5385,6 +5385,18 @@ module Cycles = struct
       )
     )
 
+  let burn env =
+    Func.share_code1 Func.Always env "cycle_burn" ("cycles", I64Type) [I64Type] (fun env get_x ->
+      Stack.with_words env "dst" 4L (fun get_dst ->
```

Comment:
We could use `2L` as word size is 64-bit and the result is a `u128` cycles value. 
Same for `msg_cycles_accept128` and `msg_cycles_refunded128` (my mistake).

```suggestion
      Stack.with_words env "dst" 2L (fun get_dst ->
```

Reply:
Nice find, please check eed383fe7c8bb30be52f15fb70ae7c536e03e53e too @luc-blaeser